images/airflow/2.9.2/bootstrap/01-root-firstpass/001-init.sh

```diff
@@ -2,3 +2,4 @@
 set -e
 
 dnf update -y
+dnf update --security -y
```

Comment:
You don't actually need to execute the update command twice, just one line of `dnf update --security -y` should be enough. From the help page:

```
  --security            Include security relevant packages, in updates
```